### ISSUE TITLE
rails tests fail after turbo-rails upgrade from 1.5.0 to 2.0.3

### ISSUE THREAD

**Issue by xeron** [hotwired/turbo-rails#587]
```
NoMethodError: undefined method `fetch' for nil:NilClass
    vendor/bundle/ruby/3.1.0/gems/actioncable-7.1.3/lib/action_cable/server/configuration.rb:38:in `pubsub_adapter'
    vendor/bundle/ruby/3.1.0/gems/actioncable-7.1.3/lib/action_cable/server/base.rb:86:in `block in pubsub'
    vendor/bundle/ruby/3.1.0/gems/actioncable-7.1.3/lib/action_cable/server/base.rb:86:in `synchronize'
    vendor/bundle/ruby/3.1.0/gems/actioncable-7.1.3/lib/action_cable/server/base.rb:86:in `pubsub'
    vendor/bundle/ruby/3.1.0/gems/actioncable-7.1.3/lib/action_cable/test_helper.rb:10:in `before_setup'
    vendor/bundle/ruby/3.1.0/gems/activerecord-7.1.3/lib/active_record/test_fixtures.rb:11:in `before_setup'
    vendor/bundle/ruby/3.1.0/gems/activesupport-7.1.3/lib/active_support/testing/setup_and_teardown.rb:40:in `before_setup'
```

**Comment by morgoth** [comment]
Most likely there is no `config/cable.yml` file in your app.

**Comment by xeron** [comment]
> Most likely there is no `config/cable.yml` file in your app.

You're right. I don't use actioncable tho and the app itself works just fine without this file.

**Comment by t27duck** [comment]
If you have `require rails/all` in your `application.rb` file or have the action cable railtie required, then action cable is loaded up. The test helpers that were added in https://github.com/hotwired/turbo-rails/commit/ef4a624de2be9b5241fe97e6c286b4bfd992fcde fires up code... somewhere, that makes action cable want to set up a server in tests which looks for `config/cable.yml`. Missing the file results in that exception.

**Comment by Manfred** [comment]
If Action Cable is now required to use Turbo in Rails, it could be good to add it to `turbo-rails.gemspec`.

**Comment by AxelTheGerman** [comment]
Also noticed this after upgrading. I don't have `require "action_cable/engine"` in my app nor `require "rails/all"`. My tests pass but when preloading the application it fails with:

```
.../vendor/bundle/ruby/3.2.0/gems/turbo-rails-2.0.5/app/channels/turbo/streams_channel.rb:34:in `<top (required)>': uninitialized constant ActionCable (NameError)
class Turbo::StreamsChannel < ActionCable::Channel::Base
```

Why would action cable be required or should the gem check if `ActionCable` is defined and only then load the relevant files?

**Comment by mishina2228** [comment]
I had the same problem, but forgot that I had previously added `require "action_cable/engine"` for #512 workaround.
Since [v2.0.6](https://github.com/hotwired/turbo-rails/releases/tag/v2.0.6) has been released with #601 included, I removed `require "action_cable/engine"` and the problem is solved.

**Comment by dhh** [comment]
Should be sorted by Action Cable no longer being required in the next release. Do verify against main and update if you see otherwise.
